### PR TITLE
Rename to camelCase

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Some more renaming to camel case of functions. Note there will be some impact on outstanding PRs with this change.

### CODE REVIEW

**Review by AaronVanGeffen — APPROVED** [review]
Big changeset, but the changes were fairly straightforward renames. Thanks, I'm happy to see this. 

Out of all the renames, I have just one comment regarding the use of `toolCancel` instead of `cancelTool`. Other than that, LGTM.

File: src/openloco/input/mouse_input.cpp [hunk 1]
```diff
@@ -283,7 +283,7 @@ namespace openloco::input
     }
 
     // 0x004CE3D6
-    void cancel_tool()
+    void toolCancel()
```
[the remark this answers is not in the document]
Comment (duncanspumpkin) [reply src/openloco/input/mouse_input.cpp:286]: To align it with all the other tool functions. Tbh it should be a member function of a tool class or something like that.

[the remark this answers is not in the document]
Comment (AaronVanGeffen) [reply src/openloco/input/mouse_input.cpp:286]: Hmm, most of the functions in the codebase tend to follow `<verb><noun>` for their names, right? With tools, we also have `isToolActive`, for example. I think that's why it stands out to me.

[the remark this answers is not in the document]
Comment (AaronVanGeffen) [reply src/openloco/input/mouse_input.cpp:286]: Yeah, I agree having a tool namespace would be good.

Anyway, we can say this comes down to personal preference. Your call.

**Pull request merged**